techniques/systemSettings/misc/clockConfiguration/4.0/metadata.xml

```diff
@@ -45,7 +40,18 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
   <SECTIONS>
     <!-- general Section -->
-    <SECTION name="Time synchronization (NTP)" component="true">
+    <SECTION name="Service status" component="true"/>
+    <SECTION name="Service restart" component="true"/>
+    <SECTION name="Package status" component="true"/>
+    <SECTION name="Time synch configuration" component="true">
+      <INPUT>
+        <NAME>SYNCH_ENABLED</NAME>
+        <DESCRIPTION>Enable NTP</DESCRIPTION>
```

Comment:
why would one not enable NTP when using NTP Tecnique ??

Reply:
Users only see its displayed name which is "Time Settings".
More over, you could want to handle clock synch and timezone without having rudder handling your ntp conf.
But still, yes, this is a hackish way to not write support for the different ntp agent availables and let the user choose if he lets us install and config NTP or not.

Reply:
Actually, the display name is the description, so "Enable NTP" - it's rather non-informative. Could you expand it a bit ?